lms/templates/bookmarks/bookmarks_list.underscore

```diff
@@ -0,0 +1,43 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header"><%= gettext("My Bookmarks") %></h2>
+
+<% if (bookmarks.length) { %>
+
+    <div class='bookmarks-results-list'>
+        <% _.each(bookmarks, function(bookmark, index) { %>
+             <a class="bookmarks-results-list-item" href="<%= bookmarkUrl(bookmark.get('course_id'), bookmark.get('usage_id')) %>" aria-describedby="bookmark-type-<%= index %> bookmark-date-<%= index %>">
```

Comment:
@muhammad-ammar I'm still not seeing an aria-labelledby on this element that references the ```<h3>```.  That completely defeats the purpose of trying to improve the length of the link text here.  Without an aria-label or aria-labelled by the screen reader will read the entire contents of the bookmark as the link text which is what we were trying to avoid.  Then, it reads the second half of it again because of the aria-describedby..  If you don't want to put the aria-labelledby on this element, the please remove the aria-describedby.